src/js/disclosure/disclosure-view.js

```diff
@@ -8,7 +8,7 @@
  *
  * @module DisclosureView
  */
-;(function(define) {
+(function(define) {
```

Comment:
Agreed. Let's leave the rule as is and just clean up the code. I'm fairly sure that r.js already handles this, so the remaining question is if any of our legacy code that bundles with Django staticfiles will be a problem. I doubt it, and as you say this is a very unlikely issue (where a page loads incomplete JS that can be fixed by a single semicolon).